spec/books/philosophy/expected_output.xhtml

```diff
@@ -1312,7 +1311,7 @@ George Edward Moore (1873&#x2013;1958)
         </div>
         <section class="summary" data-depth="1" id="auto_960c707c-79a8-44d0-bd5c-2fef4afdc553_fs-idm226171520">
           <a href="#auto_960c707c-79a8-44d0-bd5c-2fef4afdc553_0">
-            <h3 data-type="document-title" id="auto_960c707c-79a8-44d0-bd5c-2fef4afdc553_0_copy_XXX" itemprop="name">
```

Comment:
We discussed it a month ago. This attribute was added by accident in old version of recipe.